src/frontend/hooks/persistedState/usePersistedSettings.ts

```diff
@@ -4,15 +4,22 @@ import {CoordinateFormat} from '../../sharedTypes';
 
 type SettingsSlice = {
   coordinateFormat: CoordinateFormat;
+  manualCoordinateEntryFormat: CoordinateFormat;
   actions: {
     setCoordinateFormat: (coordinateFormat: CoordinateFormat) => void;
+    setManualCoordinateEntryFormat: (
+      coordinateFormat: CoordinateFormat,
+    ) => void;
   };
 };
 
 const settingsSlice: StateCreator<SettingsSlice> = (set, get) => ({
   coordinateFormat: 'utm',
+  manualCoordinateEntryFormat: 'utm',
   actions: {
     setCoordinateFormat: coordinateFormat => set({coordinateFormat}),
+    setManualCoordinateEntryFormat: coordinateFormat =>
+      set({manualCoordinateEntryFormat: coordinateFormat}),
```

Comment:
in Mapeo Mobile, we use to store an individual value in async storage that pertained to the coordinate format used specifically for manual entry. I believe the context for this is that the user's chosen coordinate format setting for _displaying_  is not necessarily the same as for _manual input_, hence the separation